alohomora/req.py

```diff
@@ -562,8 +563,11 @@ def _get_assertion(self, soup):
         LOG.debug('Pulling out SAML assertion')
         form = soup.find('form')
         input_tag = form.find('input')
-        LOG.debug('Found assertion %s', input_tag['value'])
-        return input_tag['value']
+        for inputtag in soup.find_all('input'):
```

Comment:
Updated the parsing for samlresponse. Now, we have multiple input tags in the response code